cloudinary_cli/modules/sync.py

```diff
@@ -160,6 +146,36 @@ def push(self):
         if upload_errors:
             raise Exception("Sync did not finish successfully")
 
+    def pull(self):
```

Comment:
we're returning booleans do you think we should change the name of this method to represent this?

Reply:
@strausr , in python it's not common to prefix functions with `is_` or something like that.